pkgs/development/libraries/mpich/default.nix

```diff
@@ -45,6 +57,9 @@ stdenv.mkDerivation  rec {
   '';
 
   meta = with lib; {
+    # --with-pmix silently disables gforker
+    broken = ((builtins.elem "gforker" processManagers) && pmixSupport);
+
```

Comment:
```suggestion
```
The hydra forker is also disabled, once you turn on PMIx. It seems that pmix mutually exclusive with all other PMs? I would rather add a comment to the pmixSupport option, mention that.